src/applications/vaos/tests/new-appointment/components/TypeOfFacilityPage.unit.spec.jsx

```diff
@@ -1,108 +1,108 @@
-  it('should render', () => {
-    const openFormPage = sinon.spy();
-    const updateFormData = sinon.spy();
-
-    const form = mount(
-      <TypeOfFacilityPage
-        openFormPage={openFormPage}
-        updateFormData={updateFormData}
-        data={{}}
-      />,
+import { mockFetch, resetFetch } from 'platform/testing/unit/helpers';
+import { createTestStore, renderWithStoreAndRouter } from '../../mocks/setup';
+import { fireEvent, waitFor } from '@testing-library/dom';
+import TypeOfFacilityPage from '../../../new-appointment/components/TypeOfFacilityPage';
+import { Route } from 'react-router-dom';
+import { cleanup } from '@testing-library/react';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingVSPAppointmentNew: false,
+    vaOnlineSchedulingDirect: true,
+    vaOnlineSchedulingCommunityCare: true,
+  },
+  user: {
+    profile: {
+      facilities: [
+        { facilityId: '983', isCerner: false },
+        { facilityId: '984', isCerner: false },
+      ],
+    },
+  },
+};
+
+describe('VAOS integration: VA facility page with a single-site user', () => {
+  beforeEach(() => mockFetch());
+  afterEach(() => resetFetch());
+
+  it('should show page', async () => {
+    const store = createTestStore(initialState);
+    const screen = renderWithStoreAndRouter(<TypeOfFacilityPage />, {
+      store,
+    });
+
+    expect(screen.baseElement).to.contain.text(
+      'Choose where you want to receive your care',
     );
 
-    expect(form.find('input').length).to.equal(2);
-    form.unmount();
+    expect(screen.getAllByRole('radio').length).to.equal(2);
   });
 
-  it('should not submit empty form', () => {
-    const openFormPage = sinon.spy();
-    const history = {
-      push: sinon.spy(),
-    };
-
-    const form = mount(
-      <TypeOfFacilityPage
-        openFormPage={openFormPage}
-        history={history}
-        data={{}}
-      />,
-    );
-
-    form.find('form').simulate('submit');
+  it('should show validation', async () => {
+    const store = createTestStore(initialState);
+    const screen = renderWithStoreAndRouter(<TypeOfFacilityPage />, {
+      store,
+    });
 
-    expect(form.find('.usa-input-error').length).to.equal(1);
-    expect(history.push.called).to.be.false;
-    form.unmount();
+    fireEvent.click(screen.getByRole('button', { name: /continue »/i }));
+    await expect(screen.getByRole('alert')).to.contain.text(
+      'Please provide a response',
+    );
   });
 
-  it('should call updateFormData after change', () => {
-    const openFormPage = sinon.spy();
-    const updateFormData = sinon.spy();
-    const history = {
-      push: sinon.spy(),
-    };
-
-    const form = mount(
-      <TypeOfFacilityPage
-        openFormPage={openFormPage}
-        updateFormData={updateFormData}
-        history={history}
-        data={{}}
-      />,
+  it('should show form with single required facility question, fill, and submit', async () => {
```

Comment:
This reads a little weird to me, maybe something like "should continue to the correct page based on type choice"

Reply:
Yeah that reads nicer 👍